scripts/export-unirig-onnx.py

```
@@ -0,0 +1,186 @@
+    cleaned = {(k[len("model."):] if k.startswith("model.") else k): v
+               for k, v in state.items()}            # strip the LightningModule prefix
+    missing, unexpected = model.load_state_dict(cleaned, strict=False)
+    print(f"  ckpt loaded: {len(cleaned)} tensors; missing={len(missing)} unexpected={len(unexpected)}")
+    model.eval()
+    return model, tokenizer
+
+
+def export_encoder(model, out_dir):
+    print("=== encoder (Michelangelo SAL + output_proj) ===")
+    class Enc(nn.Module):
+        def __init__(s, m): super().__init__(); s.m = m
```

Ruff (0.15.18)

[error] Multiple statements on one line (semicolon)

(E702)